### PR TITLE
Added bottomTabBar navigation + stack navigation. Added NavigationAwa…

### PR DESCRIPTION
**Pull request by Archeys**
…re interface to use as a prop interface in components for stack navigation.

### CODE REVIEW

File: package.json.orig [hunk 1]
```diff
@@ -0,0 +1,74 @@
+{
```
Comment (a-sandris) [inline package.json.orig]: kas tas par failu?

File: src/views/MoreView/Card.tsx [hunk 2]
```diff
@@ -7,10 +7,11 @@ interface CardProps {
     title: string;
     icon: IconType;
     backgroundColor: string;
+    onOpen: any;
```
Comment (a-sandris) [inline src/views/MoreView/Card.tsx]: any nebūs ok

File: src/views/index.tsx [hunk 3]
```diff
@@ -11,7 +11,7 @@ import MarkdownEvent from './MarkdownEvent';
 import UserSettings from './UserSettings';
 import { VideoView } from './VideoView';
 
-export {
+export default {
```
Comment (a-sandris) [inline src/views/index.tsx]: why?

Comment (Archeys) [reply src/views/index.tsx]: Removed

File: src/views/MoreView/Card.tsx (hunk 2 above)
Comment (Archeys) [reply src/views/MoreView/Card.tsx]: Fixed

File: package.json.orig (hunk 1 above)
Comment (Archeys) [reply package.json.orig]: Removed.

**Pull request merged**